src/Workspaces/Core/Portable/Extensions/CustomColumnInfo.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal class CustomColumnInfo
+    {
+        public (string label, string value) columnInfo;
```

Comment:
```suggestion
        public (string label, string value) ColumnInfo { get; }
```

Reply:
When I make this change I get the following error (note I changed the name to propertyInfo)

![image](https://user-images.githubusercontent.com/25776963/63276504-5cb83f00-c258-11e9-8ba8-1680eab1766f.png)

Perhaps because it's a tuple?